### ISSUE TITLE
A helper named 'percentage-columns-layout' could not be found

### ISSUE THREAD

**Issue by diogomafra** [emberjs/ember-collection#106]
I'm trying to define a collection with the helper `percentage-columns-layout`, but I'm getting the following error:

```
A helper named 'percentage-columns-layout' could not be found
```

Here is a repo that reproduces the problem (Ember 1.13.12, Ember-collection 1.0.0-alpha.5).
https://github.com/diogomafra/collection-test

**Comment by diogomafra** [comment]
It looks like the latest published version (1.0.0-alpha.5) does not have this helper. Using the master branch solves the problem.

**Comment by raytiley** [comment]
yeah... we should release an alpha 6 soon
